### PR TITLE
Switch to Scalar for OpenAPI spec UI instead @fastify/swagger-ui

### PR DESCRIPTION
**Pull request by drdaemos**
This PR switches the default documentation endpoint: `/documentation` to use Scalar (https://github.com/scalar/scalar) UI instead of outdated @fastify/swagger-ui.

Here is the example of how it looks: https://polyglot-service-main.test.lokalise.cloud/documentation

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -189,14 +197,13 @@ export async function getApp(
...
+      '/favicon.ico',
       '/login',
       '/access-token',
       '/refresh-token',
       '/documentation',
       '/documentation/json',
-      '/documentation/static/*',
-      '/documentation/static/index.html',
-      '/documentation/static/swagger-initializer.js',
+      '/documentation/@scalar/fastify-api-reference/js/browser.js',
```
Comment (kibertoad) [inline src/app.ts:206]: do we need to update e2e tests and readme accordingly?

Comment (drdaemos) [reply src/app.ts:206]: Do you have any particular places in mind?

AFAIK, docs are still at `/documentation`, but we need a js to render them.

Comment (kibertoad) [reply src/app.ts:206]: `You can access OpenAPI specification of your application, while it is running, by opening [SwaggerUI](http://localhost:3000/documentation)`

It's no longer SwaggerUI, but you are saying that the path is still the same?

See https://github.com/lokalise/node-service-template/blob/main/src/app.e2e.spec.ts, `/documentation/json` is also still valid?

Comment (drdaemos) [reply src/app.ts:206]: oh, this
yeah, I will update a link name. Path is still the same, this is a drop-in replacement. `/documentation/json` is also still valid

**Review by kibertoad — APPROVED** [review]
thank you!

**Pull request merged**